src/store/MeasurementStore.js

```diff
@@ -1,81 +1,201 @@
+                this.status = response.data.status;
+                this.error = null;
+                this.isLoading = false;
+            } catch {
+                this.error = 'Cannot pause AE measurement';
+            }
+        },
+        async stopAEMeasurement() {
+            try {
+                this.isLoading = true;
+                const response = await axios.post(
+                    config.backendUrl + '/acoustic/stop_rec'
+                );
+                this.recordingMode = response.data.recording_mode;
+                this.status = response.data.status;
+                this.error = null;
+                this.isLoading = false;
+            } catch {
+                this.error = 'Cannot stop AE measurement';
+            }
+        },
+        async startRGBMeasurement() {
+            try {
+                this.isLoading = true;
+                const response = await axios.post(
+                    config.backendUrl + '/cameras/start_rec' // TODO: change to the correct endpoint, not ready yet
+                );
+                this.error = null;
+                this.isLoading = false;
+            } catch {
+                this.error = 'Cannot start RGB Camera measurement';
+            }
+        },
+
+
+        async measureTestRGB() {
+            try {
+                this.isLoading = true;
+                const response = await axios.post(config.backendUrl + '/rgb/capturefake')
+
+                this.error = null;
+                this.loginMessage = null;
+                this.isLoading = false;
+
+                return response.data
+            } catch {
+                this.isLoading = false;
+                this.error = 'Cannot log in, wrong password! Try again.';
+            }
+        },
+
+        async getRGBPhotos() {
+            try {
+                this.isLoading = true;
+                const response = await axios.get(config.backendUrl + '/rgb-photos/how-many/')
+                this.error = null;
+                this.loginMessage = null;
+                this.isLoading = false;
+                return response.data
+            } catch {
+                this.isLoading = false;
+                this.error = 'Cannot log in, wrong password! Try again.';
+            }
+        },
+
+        async getRGBPhotosNumber() {
+            try {
+                this.isLoading = true;
+                const response = await axios.post(config.backendUrl + '/rgb-photos/how-many')
+
+                this.error = null;
+                this.loginMessage = null;
+                this.isLoading = false;
+                return response.data
+            } catch {
+                this.isLoading = false;
+                this.error = 'Cannot log in, wrong password! Try again.';
+            }
+        },
+
+        logout() {
+            this.token = null;
+            delete axios.defaults.headers.common['Authorization'];
+            localStorage.removeItem('token');
+        },
```

Comment:
Tady ty uzivatelske funkce musi byt v measurement storu?

Reply:
nemusí. to je nejaká chyba. už je to fixnuté.